src/nextjs/templates.ts

```diff
@@ -711,3 +711,139 @@ export default function RootLayout({
   )
 }
 `;
+
+export function getAiRulesFileContent(): string {
+  return `These examples should be used as guidance when configuring Sentry functionality within a project.
+
+# Exception Catching
+
+- Use \`Sentry.captureException(error)\` to capture an exception and log the error in Sentry.
+- Use this in try catch blocks or areas where exceptions are expected
+
+# Tracing Examples
+
+- Spans should be created for meaningful actions within an applications like button clicks, API calls, and function calls
+- Ensure you are creating custom spans with meaningful names and operations
```

Comment:
another idea for a rule: 

```suggestion
- Ensure you are creating custom spans with meaningful names and operations
- Span names should be parameterized. For example, when get a user by an id, name the span \`fetch /users/:id\` instead of \`fetch /users/1234\`
```

Reply:
I like this! Thanks for the add!